includes/wizards/class-reader-revenue-wizard.php

```diff
@@ -433,19 +433,29 @@ public function fetch_all_data() {
 		$stripe_data                            = Stripe_Connection::get_stripe_data();
 		$stripe_data['can_use_stripe_platform'] = Donations::can_use_stripe_platform();
 
+		$billing_fields = [];
+		if ( 'wc' === $platform && $wc_installed ) {
```

Comment:
Nit: There's a helper method `Donations::is_platform_wc()` which you could use here.

Reply:
Fixed with 0a6a4fc66a093394e052e076badfe656f8570282